db/model/Gdoc/rawToEnriched.ts

```diff
@@ -1171,3 +1176,13 @@ export function parseDetails(details: unknown): {
         parseErrors: detailsWithErrors.flatMap((detail) => detail.parseErrors),
     }
 }
+
+function parseExpandableParagraph(
+    raw: RawBlockExpandableParagraph
+): EnrichedBlockExpandableParagraph {
+    return {
+        type: "expandable-paragraph",
+        items: compact(raw.value.map(parseRawBlocksToEnrichedBlocks)),
```

Comment:
the only scenario I could get raw.value to not be an array is if `[.+expandable-paragraph]` is defined as `{.expandable-paragraph}`. Is that what you're referring to here?

Reply:
Yup, not essential. But if it happens it will break the page (with a less descriptive error) until the author fixes it versus showing up situated in context with a pink error box.

Reply:
Now handling wrong type (object vs array) and missing content:

<img width="697" alt="Screenshot 2023-05-29 at 15 54 19" src="https://github.com/owid/owid-grapher/assets/13406362/3ea4a5df-d9ad-47e0-afbe-3b413b66e433">

[Preview](http://staging-site-2204-datapages-add-collapse-functionality-to-faq-s/admin/gdocs/1fLhcFxbWSa09oM6ubryg5HcaZ4N3xsT44OKW0_7tAJY/preview)